docs/entities/renderer.md

```diff
@@ -0,0 +1,226 @@
+            // The name of the part.
+            "head",
+            // The CubeListBuilder we want to add. While it is possible to add multiple cubes into one part,
+            // it is generally discouraged and only one cube per PartDefinition should be used.
+            CubeListBuilder.create()
+                // The UV coordinates to use within the texture. Texture binding itself is explained below.
+                // In this example, we start at U=10, V=20.
+                .texOffs(10, 20)
+                // Add our cube. Again, while multiple can be added, it is recommended to only add one.
+                .addBox(
+                    // The origin of the cube, relative to the parent object's position.
+                    -5, -5, -5,
+                    // The size of the cube.
+                    10, 10, 10
+                ),
+            // An additional offset to apply to all elements of the CubeListBuilder. Besides PartPose#offset,
+            // PartPose#offsetAndRotation is also available. This can be reused across multiple PartDefinitions.
+            PartPose.offset(0, 8, 0)
+        );
+        // We can now add children to any PartDefinition, thus creating a hierarchy.
+        PartDefinition part1 = root.addOrReplaceChild(...);
+        PartDefinition part2 = head.addOrReplaceChild(...);
+        PartDefinition part3 = part1.addOrReplaceChild(...);
+        // At the end, we create a LayerDefinition from the MeshDefinition.
+        // The two integers are the expected dimensions of the texture; 64x32 in our example.
+        return LayerDefinition.create(mesh, 64, 32);
+    }
+}
+```
+
+Note that in the above example, we directly extend `EntityModel`; depending on your use case, it might be more appropriate to use one of the subclasses instead. When creating a new model, it is recommended you have a look at whatever existing model is closest to your use case, and then work from there.
+
+:::tip
+The [Blockbench][blockbench] modeling program is a great help in creating entity models. To do so, choose the Modded Entity option when creating your model in Blockbench.
+
+Blockbench also has an option to export models as a `LayerDefinition` creation method, which can be found under `File -> Export -> Export Java Entity`.
+:::
+
+### Registering a Layer Definition
+
+Once we have our entity layer definition, we also need to register it in `EntityRenderersEvent.RegisterLayerDefinitions`. To do so, we need a `ModelLayerLocation`, which essentially acts as an identifier for our layer (remember, one entity can have multiple layers).
+
+```java
+// Our ModelLayerLocation.
+public static final ModelLayerLocation MY_LAYER = new ModelLayerLocation(
+    // Should be the name of the entity this layer belongs to.
+    // May be more generic if this layer can be used on multiple entities.
+    ResourceLocation.fromNamespaceAndPath("examplemod", "example_entity"),
+    // The name of the layer itself.
+    "main"
```

Comment:
Should mention `main` should be used for the base model while other layer names should be descriptive of what it will be rendering for.